_targets.R

```diff
@@ -1,12 +1,13 @@
 library(targets)
 
 options(tidyverse.quiet = TRUE)
-tar_option_set(packages = c("tidyverse", "lubridate","rmarkdown","dataRetrieval","knitr","leaflet","sf")) 
+tar_option_set(packages = c("tidyverse", "lubridate","rmarkdown","dataRetrieval","knitr","leaflet","sf", 'purrr')) 
```

Comment:
The targets `p1_NLCD_data_zipped` and `p1_NLCD_data_zipped` build for me (I'm assuming because you used `sbtools::` notation), but you could also add `"sbtools"` to `tar_option_set` here.

Reply:
Yea probably build because sbtools is already a package you have install, but its important to ads this to the `_targets.R` file.